### DISCUSSION TITLE
拥抱开源真正的要义：那些计算机技术之外的技术 - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#68] [Announcements]
# 拥抱开源真正的要义：那些计算机技术之外的技术 - 「开源之道」

https://opensourceway.community/posts/opensource_culture/2021-oscar-culture-need-sponsor/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)

虽然已经过期, 
不过, 每年都有这个大会, 

那么, 从 FLOSS/自由|开源软件 角度来看这一文案, 就值得每年修订至少4次以上,
来决定哪种更加触动人心, 以及传递了必要和正确的态度;


去年看:

- 强调社会技术
- 经过一年的尝试, 现在看, 还是得强调基本的开发技术, 
- 如果一个社区连一个拿的出手的开源作品都没, 完全在进行各种社交活动, 那么很难让人相信, 这是个技术共同体了...

以及, 文案最关键的应该是制造出一个可以自发传播的 slogan , 否则, 和平常的文章没什么区别,

今天, 大会核心主题是什么?
应该以什么 slogan 为核心来宣传?

**Reply by lijiangsheng1** [reply]
大会也有多样性，有项目固然好，没有也没办法。努力推进，赢得信任。
